glyphs-reader/src/font.rs

```diff
@@ -323,6 +323,18 @@ impl Layer {
         })
     }
 
+    /// NOTE: panics if called on a non-bracket layer
```

Comment:
why is that desirable? Just return None or an empty map or something? Or return an error?

Reply:
it's internal API, and should only ever be called when working with things we know are bracket layers, which are already sorted (as in they're in `glyph.bracket_layers` instead of `glyph.layers`).

this is basically enforcing an invariant.